### PR TITLE
fix bug in qt.d

### PR DESCRIPTION
**Pull request by dragonwu0919**
According to issue #166 , I comment the line 76 in `qt.d` and fix the bug of script.

https://github.com/solvcon/devenv/blob/ada270f51856644ea937581072eb8a43e5c54c84/scripts/build.d/qt#L76

### CODE REVIEW

File: scripts/build.d/qt [hunk 1]
```diff
@@ -73,7 +73,7 @@ pushd ${QTSRC}/${pkgfull}/build
   #cfgcmd+=("-DBUILD_qtlanguageserver=OFF")
   #cfgcmd+=("-DBUILD_qtsvg=OFF")
   cfgcmd+=("-DBUILD_qtquicktimeline=OFF")
-  cfgcmd+=("-DBUILD_qtquick3d=OFF")
+  #cfgcmd+=("-DBUILD_qtquick3d=OFF")
```
Comment (dragonwu0919) [inline scripts/build.d/qt:76]: Add back the module qtquick3d

**Review by dragonwu0919 — COMMENTED** [review]
Add back the module qtquick3d.

**Comment by dragonwu0919** [comment]
@yungyuc I have tried to fix issue166. Please help me review the code. Thank !!

**Comment by ThreeMonth03** [comment]
It looks good to me.

**Comment by yungyuc** [comment]
> @yungyuc I have tried to fix issue166. Please help me review the code. Thank !!

Thanks, @dragonwu0919 .  This looks nice.  @j8xixo12 , could you please help review?

**Comment by j8xixo12** [comment]
Thanks @yungyuc, this change LGTM !

**Pull request merged**